internal/provider/incident_escalation_path_resource.go

```diff
@@ -127,6 +128,9 @@ func (r *IncidentEscalationPathResource) Schema(ctx context.Context, req resourc
 				Optional:            true,
 				ElementType:         types.StringType,
 				Computed:            true,
+				PlanModifiers: []planmodifier.List{
```

Comment:
Could we use `SetAttribute` / `planmodifier.Set` etc here? I don't think we care about the order of these IDs, so Set is better than List!